TLM/TLM/Manager/Impl/OptionsManager.cs

```diff
@@ -244,6 +244,8 @@ public class OptionsManager
 
             index = LoadBool(index, OptionsVehicleRestrictionsTab.NoDoubleCrossings);
             index = LoadBool(index, OptionsVehicleRestrictionsTab.DedicatedTurningLanes);
+
+            index = LoadBool(index, OptionsGeneralTab.DifferentiateDefaultSpeedsInNormalView);
```

Comment:
This `index = load(index)` code is flawed. The values are stored at fixed positions, there is no point in advancing the read pointer, and we should be loading from fixed byte offsets. See in my feature creep branch i rewrote this piece, if you'd like to take this code i'll be very happy :) https://github.com/kvakvs/TMPE/blob/u-speedlimits-tool-feature-creep/TLM/TLM/Manager/Impl/OptionsManager.cs#L74-L148

Reply:
That's a big improvement - I'll send that in as separate PR later today.

Reply:
#1321